--- node_modules/ws/lib/WebSocket.js	2017-12-10 23:49:21.000000000 -0700
+++ ../educer/node_modules/ws/lib/WebSocket.js	2019-02-14 18:46:28.587914669 -0700
@@ -179,7 +179,10 @@
     this.readyState = WebSocket.CLOSING;
     this._finalized = true;
 
-    if (typeof error === 'object') this.emit('error', error);
+    if (typeof error === 'object') {
+      // this.emit('error', error);
+      console.warn('Ignoring web socket error:', error);
+    }
     if (!this._socket) return this.emitClose();
 
     clearTimeout(this._closeTimer);
